### PR TITLE
feat(cov): add simulator tests to coverage

### PR DESCRIPTION
**Pull request by fmorency**
This PR adds simulator tests to coverage metrics. Adding this support effectively bumps the current code coverage by +18%.

@Reecepbcups Does SL have a CodeCov account I could use to upload coverage?

### CODE REVIEW

File: .github/workflows/simulator.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         run: make sim-after-import
 #      #  Requires wiring v2
 #      # https://github.com/strangelove-ventures/poa/issues/199
```
Comment (Reecepbcups) [inline .github/workflows/simulator.yml:28]: Will wait for v2 to be merged first

**Comment by Reecepbcups** [comment]
> Does SL have a CodeCov account

I do not believe so

File: .github/workflows/simulator.yml (hunk 1 above)
Comment (fmorency) [reply .github/workflows/simulator.yml]: [e64d894](https://github.com/strangelove-ventures/poa/pull/204/commits/e64d894cf642bf47c73808fb4f22e8e2e1d2fe10)

**Comment by fmorency** [comment]
> > Does SL have a CodeCov account
> 
> I do not believe so

If you get one I can add the wiring and badges to README.

**Comment by Reecepbcups** [comment]
Ill make an issue to track. Our team member who handles these things is OOO for a bit.

**Review by Reecepbcups — APPROVED** [review]

**Pull request merged**